core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaUpdateRecipesTranslator.java

```diff
@@ -214,23 +229,27 @@ public void translate(GeyserSession session, ClientboundUpdateRecipesPacket pack
             }
         }
 
-        // FIXME: if the server/viaversion doesn't send trim recipes then we shouldn't either.
```

Comment:
Preserve this comment please

Reply:
https://github.com/GeyserMC/Geyser/pull/4056/files#diff-63970a81cc5c7b300e04453fd703f6292c88875c69b956dc48ef5a7793f8bea9R234 It is preserved, no idea why it shows up that butchered